server/routes/classes.ts

```diff
@@ -27,28 +50,65 @@ this is responsible of another microservice
 
 //TODO: Connect to course microservice in frontend
 
-router.post("/", async (req: Request, res: Response) => {
-    const { title, description, order, file }: ClassInputs = req.body
-
-    if (!title || !description || !order || !file) {
-        return res.status(400).json({
-            error: "Missing required fields (title, description, order, file)",
-        })
+router.post("/", upload.single('file'), async (req: Request, res: Response) => {
+    
+    try{
+
+
+        const { title, description, order }: ClassInputs = req.body
+
+        if (!title || !description || !order || !req.file) {
+            return res.status(400).json({
+                error: "Missing required fields (title, description, order, file)",
+            })
+        }
+
+        const newClass = Class.build({
+            title,
+            description,
+            order,
+            file: 'dummy',
+        });
+
+        try{
+
+            const savedClass = await newClass.save()
+
+            //Save blob to storage
+            const blob = bucket.file(`${uuidv4()}-${req.file.originalname}`);
+            const blobStream = blob.createWriteStream({
+                metadata: {
+                    contentType: req.file.mimetype,
+                },
+            });
+
+            //Error case
+            blobStream.on('error', async (err) => {
+                console.error('Error al subir el archivo:', err);
```

Comment:
The console logs are unnecesary

Reply:
this console.error exists also in materials. I did it by similirity